package.json

```diff
@@ -30,17 +30,18 @@
         "precheck:size": "yarn gulp build:dist:min",
         "check:static": "npm-run-all --parallel lint tsc",
         "format": "prettier-eslint --write",
-        "lint": "eslint --cache --max-warnings 0 --ext .ts,.js src",
+        "lint": "jest --config=jest-lint.config.js",
         "precommit": "lint-staged",
         "gen:declarations": "tsc --p ./tsconfig.build.json",
         "build": "npm-run-all build:targets gen:declarations",
         "build:targets": "gulp build",
         "prebuild:targets": "rm -rf dist && rm -rf lib",
         "tsc": "tsc",
-        "test": "jest --config=jest.json",
+        "test": "jest --config=jest-test.config.js",
         "test:ci": "yarn test --ci",
         "test:watch": "yarn test --watch",
```

Comment:
I say we get rid of `test:watch` now? What do you think?

Reply:
I would say we could remove a few, but test:watch is only test vs validate is both test and lint.